### ISSUE TITLE
Absholutely... Thish will work!

### ISSUE THREAD

**Issue by Pellaeon83** [dalaranwow/dalaran-wow#4695]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)
http://www.wowhead.com/quest=11330/absholutely-thish-will-work
https://www.youtube.com/watch?v=ZN-XDtDdSuE

First of all, I know this bug is fixed on Algalon realm, I am just asking to apply the fix to the Kirin-Tor realm as this quest opens chains for Skorn and Gjarlebron plus few quests in Fort Wildevar, which is 20+ quests that are at the moment unobtainable because of the issue.

**Current behaviour**: Using the item whith the prisoner targeted does nothing. No credit is given.

**Expected behaviour**: When used the prisoner should drink the item stand and yell, and quest progress made. As shown in the video.

**Steps to reproduce the problem**:

1. Talk to Peppy Wrongnozzle to accept the quest, after completing Forsaken Intel- http://www.wowhead.com/quest=11328/mission-forsaken-intel
2. enter the prison, target the prisoner and use the item.
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @Pellaeon83 , your comment in #4408 was enough, it's not needed to write a new report when there is one existing already. Closing, please continue in #4408
Thank you.

**Comment by Pellaeon83** [comment]
I'm sorry, I just wasn't sure if closed issues are flagged when new comments show up, so I put a new one just in case it didn't. Again, I apologize for the trouble.
